src/v2/view-builder/ViewFactory.js

```diff
@@ -41,7 +40,7 @@ const VIEWS_MAPPING = {
     password: EnrollFactorPasswordView
   },
   'challenge-factor': {
-    email: RequiredFactorEmailView,
+    email: OTPEmailView,
```

Comment:
would rather delete `OTPEmailView` and merge code into `RequiredFactorEmailView.js`

Reply:
Will rename OTPEmailView as RequiredFactorEmailView